### ISSUE TITLE
object data view reseting 

### ISSUE THREAD

**Issue by abunajmi** [barryvdh/laravel-debugbar#1458]
I have a problem with the object data view, when I open the project page, the object data view appears but after a while the object data view resets to zero and the object views disappear, what do you think is the cause?

![image](https://github.com/barryvdh/laravel-debugbar/assets/47467302/5d3baafb-7440-4d0e-8423-bcaa9272edfb)

**Comment by abunajmi** [comment]
yes I can see it, but what can I do by looking at it, what solutions can be obtained by looking at it, anyway thanks for your's response

**Comment by erikn69** [comment]
When I say look there, I mean check the list of calls, the ajax calls replace the first one, which has the views, it seemed enough to say look there, I thought you would check it 😄😄😄

**Comment by parallels999** [comment]
Retaled https://github.com/barryvdh/laravel-debugbar/issues/1413
